src/components/Home/Ritika/Week4/Movies/SingleMovieDisplay/MovieDisplay.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import { Outlet, useNavigate, useParams } from 'react-router-dom';
+import { Movie } from '../../utils/interface';
+import { SingleMovieDisplayWrapper } from './MovieDisplay.styled';
+
+const MovieDisplay: React.FC = () => {
+  const API_URL = `http://www.omdbapi.com/?apikey=${process.env.REACT_APP_API_KEY}`;
```

Comment:
You can add this info, in Readme.md file, so that anyone can get to know that where to save the secret API key